### DISCUSSION TITLE
Old (2021) scope is suddenly gone, but its name is taken

### DISCUSSION THREAD

**Discussion by see-king** [teambit/bit#8378] [Help]
We have several components in a scope created in 2021 that we haven't accessed for about a year or so.

When installing the project from scratch, bit says it cannot fetch the data

There are **no scopes** under our account anymore, but if we try to create a scope with the same name - it says that the scope name is taken (see the screenshot)

I suspect  it's related somehow to v.14 - v.15 breaking changes. 

If the scope still exists somewhere in limbo, can it be restored? Tried installing older version of bit via bvm - it still doesn't see the remote scope.

If the scope is inaccessible, what would be the right way to export/relink the components again so that the existing projects may use them?

Thanks in advance! 

![Image 09-01-2024 at 11 58](https://github.com/teambit/bit/assets/12648914/827c6dfe-e9b0-420d-8061-66b72e7df285)

**Comment by ranm8** [comment]
Thanks for reporting! This might be a legacy scope use https://legacy.bit.cloud to access it.
